### ISSUE TITLE
Suggestion for not equal.

### ISSUE THREAD

**Issue by PumpkinSeed** [PyHP-pph/PyHP-pph#5]
Javascript using `==` as comparison operators and `===` as strict comparison operators. Can this new awesome lang using `====` for not equal? 

For example: 
```
if (1 ==== 2) {
  // true or how the if evaulate it 
}
```

**Comment by Boscillator** [comment]
The real reason `===` is bad, is that `==` still works, except it can get you into real problems in edge cases. `==` and `===` still should do something in PyHP++#

I propose that `==` returns a the string `"the values $a and $b are the same"` if they are and `"the values $a and $b are not the same"` if they are not. (with `$a` and `$b` substituted for the correct value). That way (do to type contortion), the expression will always evaluate true.

`===` should do something equally ludicrous.

**Comment by alexbuzzbee** [comment]
Perhaps `===` should return the difference between the two. `-2 === 5` would be `8`, `3 === -12` would be `16`, etc.

**Comment by alexbuzzbee** [comment]
But then how do we *actually* check if two values are equal?

**Comment by boynedmaster** [comment]
equals(val1, val2) perhaps?

**Comment by boynedmaster** [comment]
Although another problem arises if you override the functionality of ==, the original source code will end up being invalid.

**Comment by Boscillator** [comment]
That's true. Maybe we should keep `==` normal.

**Comment by DyingEcho** [comment]
I've created `operators` branch, I will add suggestions here to `operators.md`.

**Comment by DyingEcho** [comment]
What do you think of [this](https://github.com/PyHP-pph/PyHP_pph/blob/operators/spec/operators.md#syntax)?

**Comment by Kampfkarren** [comment]
Not too sure about it. I forget who said it, but I remember someone here saying something along the lines of "We want to make a bad language because it has all these different languages, not a bad language with elements from the languages thrown in. "

**Comment by ghost** [comment]
That's true, but all of the inspiration languages use `==`. PyHP++# is intended to be satirical and strange to work with, and so having only `==` would  be against the 'feel' of the language. In a way it's like the inversion of `False` and `True`.
